docs/cli/server.md

```diff
@@ -146,7 +146,7 @@ The ```bor server``` command runs the Bor client.
 
 - ```v5disc```: Enables the experimental RLPx V5 (Topic Discovery) mechanism (default: false)
 
-- ```txarrivalwait```: Maximum number of milliseconds to wait before requesting an announced transaction (default: 100)
+- ```txarrivalwait```: Maximum number of milliseconds to wait before requesting an announced transaction (default: 500)
```

Comment:
Should we change this comment also: 500->500ms?

Reply:
This is odd... this one is also pulling up an older commit somehow.  on my end it looks updated:

https://github.com/thogard785/bor/blame/fetch-timeout-config/docs/cli/server.md#L149

I'll try to figure out why the most recent commit is being overwritten. Very strange.

Reply:
It looks good in https://github.com/maticnetwork/bor/pull/734/files#diff-ca9f0d80fbfe0fad5fdb2bf71db6b22dada4751d4107ac0ca8566cbd5c7c1639R149 the file changes section

Reply:
commit that updates this line to duration:

3c23de2